wallet_presentation/src/commonMain/kotlin/net/thechance/mena/wallet/presentation/screen/export/ExportTransactionsState.kt

```diff
@@ -11,22 +11,28 @@ data class ExportTransactionsState(
     val toast: CustomToastState = CustomToastState(),
     val isCustomFilterCardSelected: Boolean = false,
     val isDownloadLoading: Boolean = false,
+    val isDownloadButtonEnabled: Boolean = true,
     val isViewAndShareLoading: Boolean = false,
     val isViewAndShareButtonEnabled: Boolean = true,
-    val isDownloadButtonEnabled: Boolean = true,
-    val hasShownEmptyFileToast: Boolean = false,
-    val noInternetConnection: Boolean = false,
-    val selectedTransactionsTypes: Set<FilterType> = emptySet(),
-    val isDateBottomSheetVisible: Boolean = false,
-    val datePickerMode: DatePickerMode = DatePickerMode.START_DATE,
-    val defaultStartDate: LocalDate? = null,
-    val defaultEndDate: LocalDate? = null,
-    val startDate: LocalDate? = null,
-    val endDate: LocalDate? = null,
+    val filterState: FilterState = FilterState(),
+    val dateState: DateState = DateState(),
     val hasNoTransactionsError: Boolean = false
 ) {
     val hasActiveFilters: Boolean
-        get() = ((startDate != null || endDate != null) && selectedTransactionsTypes.isNotEmpty())
+        get() = filterState.startDate != null && filterState.endDate != null && filterState.selectedTransactionsTypes.isNotEmpty()
```

Comment:
I think it should be like this
```suggestion
        get() =( filterState.startDate != null ||  filterState.endDate != null )&& filterState.selectedTransactionsTypes.isNotEmpty()
```

Reply:
Actually no, all filter fields should be filled with values to export a transaction